tools/src/scie_pants/install_pants.py

```diff
@@ -105,20 +105,10 @@ def install_pants_from_pex(
         try:
             ptex.fetch_to_fp(pex_url, pants_pex.file)
         except subprocess.CalledProcessError as e:
-            #  if there's only one dot in version, specifically suggest adding the `.patch`)
-            suggestion = (
-                "Pants version format not recognized. Please add `.<patch_version>` to the end of the version. For example: `2.18` -> `2.18.0`.\n\n"
-                if version.base_version.count(".") < 2
-                else ""
-            )
             fatal(
-                f"Wasn't able to fetch the Pants PEX at {pex_url}.\n\n{suggestion}"
-                "Check to see if the URL is reachable (i.e. GitHub isn't down) and if"
-                f" {pex_name} asset exists within the release."
-                " If the asset doesn't exist it may be that this platform isn't yet supported."
-                " If that's the case, please reach out on Slack: https://www.pantsbuild.org/docs/getting-help#slack"
-                " or file an issue on GitHub: https://github.com/pantsbuild/pants/issues/new/choose.\n\n"
-                f"Exception:\n\n{e}"
+                f"Wasn't able to fetch the Pants PEX at {pex_url}.\n"
+                "Check to see if the URL is reachable.\n\n"
+                f"Exception:\n{e}"
```

Comment:
Simplified error message, as we shouldn't get here unless we have a working URL already..